packages/minter-contracts/test-hs/Makefile

```diff
@@ -6,16 +6,12 @@ EXTRA_STACK_OPTIONS ?= ""
 
 all: test
 
-# Compile contracts necessary for tests.
-compile-contracts:
-	$(MAKE) -C ../ligo/src/swaps
-
 # Build Haskell dependencies.
 build-haskell-deps:
 	stack build --only-dependencies
 
 # Build and run tests (only in emulation).
-test: compile-contracts
+test:
```

Comment:
One minor thing: could you also update `README` in this folder, since instructions on running tests are now different?

Reply:
Ah yes. Good catch!

Reply:
Oh, looks like `test-hs/README` also has to be updated :sweat_smile:

Reply:
Ah, whoops! I'll send an update in a bit. Good catch!